OWNERS

```diff
@@ -8,9 +8,10 @@ approvers:
   - mrbobbytables
   - nikhita
   - parispittman
-  - spiffxp
   - committee-steering
 emeritus_approvers:
   - castrojo
   - calebamiles
   - grodrigues3
+  - spiffxp
```

Comment:
@spiffxp , Would you like to remain as an emeritus approver in the root owners file?